common/decisions/wc_culture_decisions.txt

```diff
@@ -0,0 +1,73 @@
+﻿embrace_void_elf_culture_decision = {
+	picture = "gfx/interface/illustrations/decisions/decision_realm.dds"
+	major = yes
+
+	ai_check_interval = 36
+
+	desc = embrace_void_elf_culture_decision_desc
+
+	is_shown = {
+		is_landed = yes
+		can_become_void_elf_trigger = yes
+		NOT = {
+			is_target_in_global_variable_list = {
+				name = unavailable_unique_decisions
+				target = flag:embrace_void_elf_culture_decision
+			}
+		}
+	}
+
+	is_valid = {
+		custom_description = {
+			text = capital_in_quelthalas_region_tooltip
+			capital_province = {
+				geographical_region = world_eastern_kingdoms_quelthalas
+			}
+		}
+		NOT = { culture = culture:void_elf }
+		culture = { has_cultural_pillar = heritage_highborne }
+		has_trait = being_void
```

Comment:
there is no way to become  a void being in the game right now so this decision cannot be taken in the normal course of the game without debug mode. this needs to be fixed, otherwise this PR needs to be shelved until infusions are done.